app/main.py

```diff
@@ -1,2 +1,67 @@
+from copy import deepcopy
+
+
 class Dictionary:
-    pass
+    def __init__(self):
+        self.capacity = 8
+        self.threshold = int(self.capacity * 2 / 3)
+        self.table = [[] for _ in range(self.capacity)]
+        self.length = 0
+
+    def __getitem__(self, key):
+        hash_key = hash(key)
+        get_ind = hash_key % self.capacity
```

Comment:
`get_ind` is something like `get_index`? I think that if you add 2 letters, it will look more readable
because if you shortened like that, then 2 letters is a minor reduction
WDYT?